Task 7/script.js

```diff
@@ -10,3 +10,6 @@ const audi = {
   year: 2005,
   color: 'white',
 };
+
+const showObjectKeys = Object.keys(audi)
```

Comment:
Reikia sukurti funkcija kuri priimtu objekta ir grazintu to objekto keys ir galetu buti pernaudojama